media-fonts/joypixels/joypixels-6.6.0.ebuild

```diff
@@ -0,0 +1,39 @@
+# Copyright 2021 Gentoo Authors
```

Comment:
Remember to bump this. We recommend using an automated tool to do bumping, e.g. repoman or pkgdev.